ui/src/components/ChipComponent/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Chip from '@material-ui/core/Chip';
+import { withStyles } from '@material-ui/core/styles';
+
+@withStyles(() => ({
+  chipLabel: {
+    whiteSpace: 'nowrap',
+    userSelect: 'text',
+    cursor: 'inherit',
+  },
+  chipRoot: {
+    whiteSpace: 'nowrap',
+    userSelect: 'text',
+    cursor: 'inherit',
+  },
+}))
+export default class ChipComponent extends Component {
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <Chip
+        classes={{ label: classes.chipLabel, root: classes.chipRoot }}
+        label={this.props.label}
```

Comment:
If I just use ```{...this.props}``` here, I am still not able to copy paste.
So instead, I had to add ```label={this.props.label}``` and ```className={this.props.className}``` after which, I believe, things worked just fine. (Except for a lot of errors inside the console) @helfi92